src/python/create.py

```diff
@@ -189,11 +189,11 @@ def create_materials(parsed_materials, project, data, citations):
 
             try:
                 # try to get the material using its name
-                name_ = parsed_material["name"]["value"]
-                newProject = cript.Project.get(
+                mat_name = parsed_material["name"]["value"]
```

Comment:
I would not have known what `mat_name` if it wasn't for the comment. How come we didn't go with `material_name` ?